meta-resin-common/recipes-core/systemd/systemd/resin_update_state_probe

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -o errexit
+set -o pipefail
+
+dev=$1
+
+# Only when root is mounted we can differenciate
+ROOT_MOUNTED=$(findmnt --output LABEL | grep resin-root | wc -l)
```

Comment:
We add not a until-linux dependency because of findmnt, right?

Reply:
I'm not sure what you mean here

Reply:
I edited by strange comment above. What I'm saying is that until-linux is a dependency of the package which includes this script, right?

Reply:
added